src/main/java/com/denizenscript/depenizen/bukkit/properties/worldguard/WorldGuardPlayerExtensions.java

```diff
@@ -0,0 +1,144 @@
+package com.denizenscript.depenizen.bukkit.properties.worldguard;
+
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.MapTag;
+import com.denizenscript.denizencore.tags.TagRunnable;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import com.denizenscript.denizencore.utilities.debugging.SlowWarning;
+import com.denizenscript.denizencore.utilities.debugging.Warning;
+import com.denizenscript.depenizen.bukkit.bridges.WorldGuardBridge;
+import com.sk89q.worldedit.bukkit.BukkitAdapter;
+import com.sk89q.worldguard.WorldGuard;
+import com.sk89q.worldguard.bukkit.WorldGuardPlugin;
+import com.sk89q.worldguard.protection.flags.Flag;
+import com.sk89q.worldguard.protection.flags.Flags;
+import com.sk89q.worldguard.protection.flags.StateFlag;
+import com.sk89q.worldguard.protection.regions.RegionQuery;
+
+public class WorldGuardPlayerExtensions {
+
+    public static Warning worldguardCanBuild = new SlowWarning("worldguardCanBuild", "The tag 'PlayerTag.worldguard.can_build' from Depenizen/Worldguard is deprecated: use 'PlayerTag.worldguard_can_build'");
```

Comment:
Should use the Deprecations.pointlessSubTagPrefix

Reply:
uh do I need to register it in denizen-core? pointlessSubTagPrefix is private